Gemfile.lock

```diff
@@ -388,6 +388,8 @@ GEM
     nokogiri (1.13.9)
       mini_portile2 (~> 2.8.0)
       racc (~> 1.4)
+    nokogiri (1.13.9-aarch64-linux)
```

Comment:
This was updated because I ran the command: ```docker compose run --rm app bundle lock --add-platform aarch64-linux```
I am using Mac M1 and docker was throwing an error because of wrong dependencies platform